src/resources/views/columns/boolean.blade.php

```diff
@@ -2,13 +2,21 @@
 <td data-order="{{ $entry->{$column['name']} }}">
 	@if ($entry->{$column['name']} === true || $entry->{$column['name']} === 1 || $entry->{$column['name']} === '1')
         @if ( isset( $column['options'][1] ) )
-            {{ $column['options'][1] }}
+            @if ( isset( $column['options']['unesc'] )  && $column['options']['unesc'])
```

Comment:
I say we drop the conditional and use ```{!! !!}``` all the way in this column type. No security concern here.